src/rdkafka_conf.c

```diff
@@ -739,6 +739,22 @@ static const struct rd_kafka_property rd_kafka_properties[] = {
           _RK(ssl.keystore_location),
           "Path to client's keystore (PKCS#12) used for authentication."
         },
+#if OPENSSL_VERSION_NUMBER >= 0x10100000
```

Comment:
This is handled differently now, see _UNSUPPORTED_SSL and add a new one _UNSUPPORTED_SSL_1_1